main.star

```diff
@@ -10,6 +10,10 @@ cdk_erigon_package = "./cdk_erigon.star"
 databases_package = "./databases.star"
 deploy_zkevm_contracts_package = "./deploy_zkevm_contracts.star"
 ethereum_package = "./ethereum.star"
+optimism_package = (
+    "github.com/leovct/optimism-package/main.star@feat/deploy-to-external-l1"
```

Comment:
Use the official repository once the feature to deploy the OP package to an external L1 will be merged